terminology/codesystem/GbaCodeSystemStorageTemperature.json

```diff
@@ -5,30 +5,33 @@
   "name": "GbaCodeSystemStorageTemperature",
```

Comment:
GBA?

Reply:
Some resources still use the old names. I plan to do re-name and update all URLs once the profiles are feature-complete.